bin/generate-nav

```diff
@@ -1,19 +1,29 @@
 #!/usr/bin/env node
 
+const {docPagesDirectory, versions} = require('../constants');
 const fileProcessor = require('../src/lib/fileProcessor');
-const { readFileSync, writeFile } = require('fs');
+const { readFileSync, writeFile, readdirSync } = require('fs');
```

Comment:
CS? (we need to make a choice :sweat_smile:)

Reply:
yup you're right, this need a linter even...
won't be in this PR though.

Reply:
oh wait there is one, maybe just lacking a rule.

Reply:
I go for no spaces, I'll what's recommended later.

Reply:
Nice, thank you!